portal-web/test/functional/com/liferay/portalweb/macros/PortalSettings.macro

```diff
@@ -653,6 +653,21 @@ definition {
 		PortalSettings.configureCurrentLanguagesCP(defaultPortalLanguage = "${newLanguage}");
 	}
 
+	macro setImportIntervalTime {
+		Variables.assertDefined(parameterList = "${importIntervalTime}");
+
+		PortalSettings.gotoConfiguration(
+			configurationCategory = "LDAP",
+			configurationName = "Import",
+			configurationScope = "Virtual Instance Scope");
```

Comment:
Don't we have some direct url to access this portlet?

Reply:
This is the macro which is widely used. I did not found any eligible existing macro for this and the url is quite long to just copy into the code